Sources/SQL/Serialization/SQLSerializer+GroupBy.swift

```diff
@@ -0,0 +1,21 @@
+extension SQLSerializer {
+    /// See SQLSerializer.serialize(groupBys:)
+    public func serialize(groupBys: [DataGroupBy]) -> String {
```

Comment:
this signature needs to appear in the protocol decl as well so it can be overridden

Reply:
done